src/plugins/intel_gpu/include/intel_gpu/runtime/layout.hpp

```diff
@@ -340,7 +340,7 @@ struct layout {
         if (format == cldnn::format::custom) {
             auto bytes_of_layout = (ov::element::Type(data_type).bitwidth() * get_linear_size() + 7) >> 3;
             auto desc_size = format.traits().desc_size;
-            OPENVINO_ASSERT(desc_size > 0, "[GPU] Invalid layout descriptor size: ", desc_size);
+            OPENVINO_ASSERT(desc_size >= 0, "[GPU] Invalid layout descriptor size: ", desc_size);
```

Comment:
This was added by https://github.com/openvinotoolkit/openvino/pull/30606/files 
and IIRC, the pr was to fix onednn cache loading when desc_size is 0 as unexpectedly. Why size 0 is allowed? (fyi @davidsnam-intel @e-ddykim )

Reply:
I'll revert this. I added because of unit-test failure. I agree this assert is no problem, but need to modify unit-test condition(missed desc_size) in the test case.